src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private final UserService userService;
+
+    public AuthenticationService(UserService userService) {
+        this.userService = userService;
+    }
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        if (user == null) {
```

Comment:
```suggestion
```

Reply:
not fixed

Reply:
not fixed

Reply:
you make this validation on 29th line, so this check is redundant